### ISSUE TITLE
Data Exchange, DE Wrapper - Impliment a single way of working with CAD files

### ISSUE THREAD

**Issue by dpasukhi** [tpaviot/pythonocc-core#1199]
OCC DE Wrapper was released with 7_7_0 OCCT.
You can check updated documentation for ver7_7_1.
All projects that exist to work with CAD files using OCCT, including commercial ones, support DE_Wrapper.
In upcoming releases, every DE library will be converted into plugins which can be plugged in and used abstractly via DE_wrapper.
I see this as a great benefit for use including python wrapping.
DE_Wrepper is currently in beta version. A full release is planned for 7_7_3 or 7_8_0.

**Comment by tpaviot** [comment]
yep, saw this module coming, I conducted a few tests for DE_Wrapper to be supported by pythonocc, without success so far
